### PR TITLE
Issue #12

### LINKED ISSUE #12: Add ESLint
Add ESLint with AirBnB style.

### PR DESCRIPTION
**Pull request by murilodag**
Fixes #12 

This isn't ready yet.
I think it is easier to review individual commits for changes so far.
Missing: 

- merge #26 to master
- merge #26 to this branch
- review missing issues (need some input from @johnpaulada)

### CODE REVIEW

**Comment by johnpaulada** [comment]
Hey @murilodag I forgot to update this. It should be Standard style, not AirBnB style. Sorry 😢 
Can you change it to use the [Standard style for ESLint](https://github.com/standard/eslint-config-standard) instead?

**Comment by johnpaulada** [comment]
Hey @murilodag I forgot to update this. It should be Standard style, not AirBnB style. Sorry 😢
Can you change it to use the Standard style for ESLint instead?
Thank you so much! 😄

**Comment by johnpaulada** [comment]
About #26 -- I'll merge it when the suggested changes are made. No worries, take your time. 😄

**Comment by johnpaulada** [comment]
I merged #26 . There seem to be conflicts with this PR.

**Comment by murilodag** [comment]
I was expecting the conflicts, I will resolve them.

**Comment by johnpaulada** [comment]
Thank you as always. :smile:

**Comment by murilodag** [comment]
Just to confirm eslint rules: eslint:recommended instead of airbnb-base, right?
https://eslint.org/docs/rules/

**Comment by johnpaulada** [comment]
Um, use [eslint-config-standard](https://github.com/standard/eslint-config-standard). The instructions should be there on the link. The `extends` key on `.eslintrc` should be "standard" not "airbnb-base" after following the installation instructions of [eslint-config-standard](https://github.com/standard/eslint-config-standard).
Thanks!

File: src/basic/basic.js [hunk 1]
```diff
@@ -34,19 +34,19 @@ function squaredSum (numbers) {
...
-function sumOfSquaresOfDifference (numbers) {
-  const mean_val = mean(numbers)
+function sumOfSquaresOfDifference(numbers) {
+  const meanVal = mean(numbers);
 
   return numbers.reduce((acc, number) => {
-    const difference = number - mean_val
-    const squared = Math.pow(difference, 2)
+    const difference = number - meanVal;
+    const squared = square(difference);
```
Comment (murilodag) [inline src/basic/basic.js]: @johnpaulada , could I remove this variable "squared" or use it on the line below?

File: src/inference/inference.js [hunk 2]
```diff
@@ -33,20 +33,20 @@ function getMM ({squaredSum, sumSquared, size}) {
...
-  const diffSquaredSum = sum(diffSquareds)
-  const diffSumSquared = Math.pow(diffSum, 2)
-  const numerator = diffSum / sampleSize
-  const rootNumerator = diffSquaredSum - diffSumSquared / sampleSize
-  const rootDenominator = sampleSize * (sampleSize - 1)
-  const denominator = Math.sqrt(rootNumerator / rootDenominator)
-  const t = numerator / denominator
-
-  return t
+function pairedTValue(numbers, twoTailed = true) {
```
Comment (murilodag) [inline src/inference/inference.js]: @johnpaulada, do you know where should this variable "twoTailed" be used?

File: src/inference/inference.js [hunk 3]
```diff
@@ -56,35 +56,35 @@ function pairedTValue (numbers, twoTailed = true) {
...
-  group2.mm = getMM(group2)
-
-  const numerator = group1.mean - group2.mean
-  const rootTerm1 = (group1.mm + group2.mm) / (group1.size + group2.size - 2)
-  const rootTerm2 = 1 / group1.size + 1 / group2.size
-  const denominator = Math.sqrt(rootTerm1 * rootTerm2)
-  const t = numerator / denominator
-
-  return t
+function independentTValue(numbers, twoTailed = true) {
```
Comment (murilodag) [inline src/inference/inference.js]: @johnpaulada, do you know where should this variable "twoTailed" be used?

**Review by murilodag — COMMENTED** [review]
I think it is done now. =)

**Comment by johnpaulada** [comment]
Great job @murilodag ! Thanks! I'll merge it now. 😄 🎉

**Pull request merged**